claim/netdata-claim.sh.in

```diff
@@ -142,6 +142,10 @@ NETDATA_USER=$(get_config_value netdata global "run as user")
 # get the MACHINE_GUID by default
 if [ -r "${MACHINE_GUID_FILE}" ]; then
         ID="$(cat "${MACHINE_GUID_FILE}")"
+        MGUID=$ID
+else
+        echo >&2 "netdata.public.unique.id is not generated yet. Please run agent at least once before attempting to claim. Agent generates this file on first startup."
+        exit 22
```

Comment:
Take a look at the exit codes at the beginning of the file. It should be documented and also check how the agent code handles the return codes as well. You should check for any collisions or other problems.

Reply:
you are right, thx